var/spack/repos/builtin/packages/namd/package.py

```diff
@@ -42,7 +42,7 @@ class Namd(MakefilePackage, CudaPackage):
     # Handle change in python-config for python@3.8:
     patch('namd-python38.patch', when='interface=python ^python@3.8:')
 
-    depends_on('charmpp@6.10.1:', when="@2.14:")
+    depends_on('charmpp@6.10', when="@2.14:")
```

Comment:
This includes `6.10.0` which was explicitly excluded before:
```suggestion
    depends_on('charmpp@6.10.1:6', when="@2.14:")
```